src/containers/main/ShellOfSecrets/SoSWidget/segments/Timer/Timer.tsx

```diff
@@ -37,12 +38,38 @@ export default function Timer() {
 
             <TimerColumn>
                 <NumberGroup>
-                    <Number>{reminingTime.days}</Number>
+                    <Number>
+                        <NumberFlow
+                            value={reminingTime.days}
+                            willChange={true}
+                            continuous={true}
```

Comment:
not sure if you need to add another package but `continuous` got moved? causing these build errors 

https://github.com/barvian/number-flow/blob/main/packages/react/CHANGELOG.md#050